_posts/2018-11-19-announcing-riff-0-2-0.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: single
+title: "Announcing riff v0.2.0"
+header:
+  overlay_image: /images/geese.jpg
+excerpt:
+  With buildpacks everywhere, smarter detection, and updated Knative support
+permalink: /blog/announcing-riff-0-2-0/
+---
+
+We are happy to announce the release of [riff v0.2.0](https://github.com/projectriff/riff/releases/tag/v0.2.0). Thank you once again, all riff and Knative contributors.
+
+The riff CLI can be downloaded from our [releases page](https://github.com/projectriff/riff/releases/tag/v0.2.0) on GitHub. Please follow one of the [getting started](/docs) guides, to create a new cluster on GKE or minikube.
+
```

Comment:
Add a quick summary of deprecations and breaking changes

- no more invoker builds
  - or custom invokers 😢, for the moment anyway 
- invoker type no longer required in `function create`
- `dev.local`
- ...what am I missing?